src/Console/ConsoleInputOption.php

```diff
@@ -113,7 +120,8 @@ public function __construct(
         array $choices = [],
         bool $multiple = false,
         bool $required = false,
-        ?string $prompt = null
+        ?string $prompt = null,
+        string $separator = ','
```

Comment:
This looks like a BC for me.

Reply:
With default it still should target 5.next

Reply:
Yes, add `,` by default in separator could be a breaking change when choices have `,` in words.

I can try to set it null by default and don't do explode if not intentionally sets.

Reply:
What about

        $x = $args->getArrayArgument('argument-list');
        $y = $args->getArrayOption('option-list');

Do we want to use them for this?
So also PHPStan will be happy here?

Reply:
i'm lost :)

Reply:
How are the array args being read inside a command?
No chance necessary here?

Reply:
ok, it's another topic. Not about the breaking change.
I will check that.